addon/ng2/models/webpack-build-common.ts

```diff
@@ -0,0 +1,92 @@
+import * as webpack from 'webpack';
+import {LoaderConfig} from '../utilities/ts-path-mappings-webpack-plugin';
+
+const path = require('path');
+const ForkCheckerPlugin = require('awesome-typescript-loader').ForkCheckerPlugin;
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+export const getWebpackCommonConfig = function(projectRoot: string) {
+  const awesomeTypescriptLoaderConfig: LoaderConfig | any = {
```

Comment:
`any` is to handle the extra `awesome-typescript-loader` config parameters. `LoaderConfig` interfaces the compilerOptions. This prevents me from having to set the following in tsconfig.json in the users project:

```
{
  "awesomeTypescriptLoaderOptions": {
    "useForkChecker": true
  }
}
```

There are a slue of other compiler options that have been added so instead of writing the entire interface, any was used. Thoughts? I can write the full interface and swap it out.